tests/network/vmi_networking.go

```diff
@@ -821,6 +821,7 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 
 			It("[outside_connectivity]should be able to reach the outside world [IPv6]", func() {
 				libnet.SkipWhenClusterNotSupportIpv6()
+				checks.SkipIfARM64(testsuite.Arch, "The Arm64 e2e test infra doesn't configure IPv6 networks, skip the test for now.")
```

Comment:
We should introduce label and skip on it

Reply:
> ARM64NotSupported

I will use the tag ARM64InfraNotSupported to prevent any misunderstanding, as the test is being skipped due to the lack of Arm64 e2e infrastructure support for IPv6 network. How's your idea?

Reply:
Good point about the ARM vs Infra issue, I agree